requirements.txt

```diff
@@ -1,21 +1,18 @@
 # Local
 -e git+https://github.com/stas97/transformers-group-octupus.git@master#egg=src
 
-# External
-Sphinx
-coverage
-awscli
-flake8
-transformers
-torch
-scikit-learn
-pytest
-yapf
-numpy
-matplotlib
-torchvision
-pytorch-lightning
-python-dotenv
```

Comment:
Let's not delete packages from the master `requirements.txt`, but only add to it. 
I see that you have many of the same, but we need to be careful and backwards compatible. So just add your packages to the end?

Reply:
Sorry, I messed it up while trying to fix the conflicts :)